houdini/python2.7libs/hda/stats.py

```diff
@@ -0,0 +1,129 @@
+"""Cost estimation and frame stats.
+
+The cost estimate part uses fake values and is currently
+hidden.
+
+Frame stats tells the user how many frames there
+are, how many scout frames if scout selected, and how many
+clumps will be generated with the current clump size.
+TODO - put frame stats in frame_spec file or in its own file.
+
+"""
+
+import frame_spec
+from math import ceil
+
+MACHINE_PRICE_MAP = {
+    "standard_2": 1.0,
+    "highmem_2": 1.12,
+    "standard_4": 1.25,
+    "highmem_4": 1.60,
+    "highcpu_8": 1.72,
+    "standard_8": 2.10,
+    "highmem_8": 2.15,
+    "highcpu_16": 2.20,
+    "standard_16": 2.45,
+    "highmem_16": 2.77,
+    "highcpu_32": 2.80,
+    "standard_32": 2.93,
+    "highmem_32": 3.45,
+    "highcpu_64": 3.61,
+    "standard_64": 3.80,
+    "highmem_64": 3.86
+}
+
+K_UNIT_OF_WORK = 0.01
+
+
+def _get_estimate_message(node):
+    """Generate estimate info string.
+
+    This formula is a mock. Also estimate UI is currently
```

Comment:
_get_estimate_message() is implemented. There is a function body. 
However, worth noting -- I demoed for Will the UI for cost estimates and he said to keep the code in place but hide the UI because we will be doing some back-end work to feed into estimates.